tests/test_messaging.py

```diff
@@ -0,0 +1,180 @@
+     configs.testpath.TEST_USER_DATA / 'user_account_two')
+])
+def test_group_chat(multiple_instance, user_data_one, user_data_two, user_data_three):
+    user_one: UserAccount = constants.user_account_one
+    user_two: UserAccount = constants.user_account_two
+    user_three: UserAccount = constants.user_account_three
+    members = [user_two.name, user_three.name]
+    main_window = MainWindow()
+    messages_screen = MessagesScreen()
+
+    with multiple_instance() as aut_one, multiple_instance() as aut_two, multiple_instance() as aut_three:
+        with step(f'Launch multiple instances with authorized users {user_one.name} and {user_two.name}'):
+            for aut, account in zip([aut_one, aut_two, aut_three], [user_one, user_two, user_three]):
+                aut.attach()
+                main_window.wait_until_appears(configs.timeouts.APP_LOAD_TIMEOUT_MSEC).prepare()
+                main_window.authorize_user(account)
+                main_window.hide()
+
+        with step(f'User {user_two.name}, get chat key'):
+            aut_two.attach()
+            main_window.prepare()
+            profile_popup = main_window.left_panel.open_user_canvas().open_profile_popup()
+            chat_key = profile_popup.chat_key
+            profile_popup.close()
+            main_window.hide()
+
+        with step(f'User {user_one.name}, send contact request to {user_two.name}'):
+            aut_one.attach()
+            main_window.prepare()
+            settings = main_window.left_panel.open_settings()
+            messaging_settings = settings.left_panel.open_messaging_settings()
+            contacts_settings = messaging_settings.open_contacts_settings()
+            contact_request_popup = contacts_settings.open_contact_request_form()
+            contact_request_popup.send(chat_key, f'Hello {user_two.name}')
+            main_window.hide()
+
+        with step(f'User {user_two.name}, accept contact request from {user_one.name}'):
+            aut_two.attach()
+            main_window.prepare()
+            settings = main_window.left_panel.open_settings()
+            messaging_settings = settings.left_panel.open_messaging_settings()
+            contacts_settings = messaging_settings.open_contacts_settings()
+            contacts_settings.accept_contact_request(user_one.name)
+            main_window.hide()
+
+        with step(f'User {user_three.name}, get chat key'):
+            aut_three.attach()
+            main_window.prepare()
+            profile_popup = main_window.left_panel.open_user_canvas().open_profile_popup()
+            chat_key = profile_popup.chat_key
+            profile_popup.close()
+            main_window.hide()
+
+        with step(f'User {user_one.name}, send contact request to {user_three.name}'):
+            aut_one.attach()
+            main_window.prepare()
+            settings = main_window.left_panel.open_settings()
```

Comment:
Looks like we need to join all lines from 75 to 79 in one method - send_contacts_request. what do you think?
In SettingsScreen for example